### ISSUE TITLE
Export package.json

### ISSUE THREAD

**Issue by msotnikov** [d3/d3#3608]
Hi.

### The problem described there
https://github.com/nodejs/node/issues/33460

### Related issues:
https://github.com/d3/d3/issues/3556
https://github.com/d3/d3/issues/3532

### Related PR with fix
https://github.com/d3/d3/pull/3576
<img width="737" alt="Снимок экрана 2022-12-05 в 16 21 34" src="https://user-images.githubusercontent.com/16710810/205659876-30bdaa5a-b338-42fc-b2c6-32dda9b92f2d.png">

### To make sure it doesn't break anything:
- NodeJS Documentation https://nodejs.org/api/packages.html#conditional-exports
- And success story :-) https://github.com/uuidjs/uuid/issues/444#issuecomment-631663778

Could you apply this, please?

**Comment by msotnikov** [comment]
fix with patch-package `patches\d3+7.7.0.patch`

```
diff --git a/node_modules/d3/package.json b/node_modules/d3/package.json
index 104c495..6ce3f02 100644
--- a/node_modules/d3/package.json
+++ b/node_modules/d3/package.json
@@ -30,8 +30,11 @@
   "jsdelivr": "dist/d3.min.js",
   "unpkg": "dist/d3.min.js",
   "exports": {
-    "umd": "./dist/d3.min.js",
-    "default": "./src/index.js"
+    ".": {
+      "umd": "./dist/d3.min.js",
+      "default": "./src/index.js"
+    },
+    "./package.json": "./package.json"
   },
   "dependencies": {
     "d3-array": "3",

```

**Comment by msotnikov** [comment]
?
